gtda/diagrams/tests/test_features_representations.py

```diff
@@ -90,6 +90,15 @@ def test_nop_transform(n_jobs):
     assert_almost_equal(nop.fit_transform(X), diagram_res)
 
 
+@pytest.mark.parametrize('n_jobs', [1, 2, -1])
+def test_atol_transform(n_jobs):
+    atol = ATOL(quantiser_params={'n_clusters': 2, 'random_state':0},
+                n_jobs=n_jobs)
+    diagram_res = np.array([[1.01831564, 1.01831564, 1.01831564, 1.01831564]])
+
+    assert_almost_equal(atol.fit_transform(X), diagram_res)
```

Comment:
More tests are needed:
- [ ] list of `n_clusters`,
- [ ] one of the diagrams in the collection (for a single dimension), has only trivial points.